tests/requirements.txt

```diff
@@ -1,4 +1,4 @@
-git+https://github.com/jina-ai/jina.git@v2.0.21#egg=jina[standard]
+git+https://github.com/jina-ai/jina.git@b0e46a3027a24b7e251085c13953b2ad38034a53#egg=jina[standard] # includes a fix for handling DA.extend(None)
```

Comment:
Can this be changed back to a proper Jina version again?

Reply:
done